### PR TITLE
blender: add page

### PR DESCRIPTION
**Pull request by waldyrious**
Mostly based on https://www.blender.org/manual/advanced/command_line/arguments.html
with some additional help from http://blender.stackexchange.com/questions/3061/setting-render-options-from-the-command-line

### CODE REVIEW

**Comment by waldyrious** [comment]
It worked 👍 thanks @ostera :P

File: pages/common/blender.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# blender
+
+> Command-line interface to the Blender 3D computer graphics application.
+> Arguments are executed in the order they are given.
+
+- Render all frames of an animation in the background (without UI):
+
+`blender -b {{file.blend}} -a`
```
Comment (agnivade) [inline pages/common/blender.md]: Suggest keeping `.blend` out of the token wrapper.

File: pages/common/blender.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Render an animation to a specific output path relative to the .blend file:
+
+`blender -b {{file.blend}} -o //{{render/frame_###.png}} -a`
```
Comment (agnivade) [inline pages/common/blender.md]: What is the `//` for ?

File: pages/common/blender.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Render the 10th frame of an animation as a single image:
+
+`blender -b {{file.blend}} -f {{10}}`
+
+- Render the second last frame in an animation as a JPEG image:
+
+`blender -b {{file.blend}} -F JPEG -f -2`
```
Comment (agnivade) [inline pages/common/blender.md]: Shouldn't JPEG and -2 be inside tokens ?

File: pages/common/blender.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Render all frames of an animation in the background (without UI):
+
+`blender -b {{file.blend}} -a`
+
+- Render an animation to a specific output path relative to the .blend file:
+
+`blender -b {{file.blend}} -o //{{render/frame_###.png}} -a`
+
+- Render the 10th frame of an animation as a single image:
```
Comment (agnivade) [inline pages/common/blender.md]: Where is the image rendered ? In a separate window or to a file ?

File: pages/common/blender.md (hunk 2 above)
Comment (waldyrious) [inline pages/common/blender.md]: It's to activate relative path processing (relative to the source .blend file). Without it the path is assumed to be absolute. I believe the relative option is more useful, but let me know what you think.

File: pages/common/blender.md (hunk 1 above)
Comment (waldyrious) [inline pages/common/blender.md]: Sure.

File: pages/common/blender.md (hunk 4 above)
Comment (waldyrious) [inline pages/common/blender.md]: Oh, good point. I suppose all these invocations should have output paths. The default is saving to /tmp, which is not intuitive IMO.

**Comment by waldyrious** [comment]
@agnivade please take another look.

**Comment by agnivade** [comment]
:shipit:

**Comment by waldyrious** [comment]
@ostera care to take a look?

**Comment by agnivade** [comment]
@ostera - ping :)

**Comment by waldyrious** [comment]
@agnivade since @ostera didn't leave any comment about the command itself, I guess it's ok to merge with the thumbs-up from the two of us. LGTM.

**Comment by waldyrious** [comment]
@slashme I meant to ping you when I first planned to make this PR, but the git mess-up above made me forget to do it. I'd like to hear your thoughts, as a blender user, about the tldr page :)

**Comment by slashme** [comment]
I think you've covered the most important options that the widest group of users will need very well. Good job!

**Pull request merged**